android/res/values/colors.xml

```diff
@@ -86,6 +86,13 @@
   <color name="button_accent_text_disabled">#42000000</color>
   <color name="button_accent_text_disabled_night">#4CFFFFFF</color>
 
+  <color name="button_red">@color/button_red_normal</color>
+  <color name="button_red_normal">@color/base_red</color>
```

Comment:
А, base_red и button_red переглядел, сори.


> On Aug 2, 2016, at 11:03 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/values/colors.xml <https://github.com/mapsme/omim/pull/3905#discussion_r73226639>:
> 
> > @@ -86,6 +86,13 @@
> >    <color name="button_accent_text_disabled">#42000000</color>
> >    <color name="button_accent_text_disabled_night">#4CFFFFFF</color>
> >  
> > +  <color name="button_red">@color/button_red_normal</color>
> > +  <color name="button_red_normal">@color/base_red</color>
> Не перекрёстные. Расцветка для красной кнопки.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/3905/files/5ba7c2e922344823b5f94408099497dfb6332cb2..e8b028a5e270e8dff08c7f290630da6dc0828a32#r73226639>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACKTRApejVko2cyTOf-a-qEw1Fp7SNKVks5qb6KvgaJpZM4JZDqJ>.
>